Makefile

```diff
@@ -15,23 +15,24 @@ include ../phoenix-rtos-build/Makefile.common
 SYSROOT := $(shell $(CC) $(CFLAGS) -print-sysroot)
 MULTILIB_DIR := $(shell $(CC) $(CFLAGS) -print-multi-directory)
 LIBC_INSTALL_DIR := $(SYSROOT)/lib/$(MULTILIB_DIR)
-LIBC_INSTALL_NAMES := libc.a libm.a crt0.o libg.a libpthread.a
+LIBC_INSTALL_NAMES := libc.a libm.a libg.a libpthread.a
 HEADERS_INSTALL_DIR := $(SYSROOT)/usr/include
 LIBNAME := libphoenix.a
 
 ifeq (,$(filter-out /,$(SYSROOT)))
 $(error SYSROOT is not supported by the toolchain. Use cross-toolchain to compile.)
 endif
 
-# libphoenix shoudl be linked only with GCC library therefore LDLIBS is overloaded
-LDLIBS = $(GCCLIB)
 CFLAGS += -Iinclude -fno-builtin-malloc
 
 
-OBJS = $(PREFIX_O)_startc.o
+OBJS :=
+# crt0.o should have all necessary initialization + call to main()
+CRT0_OBJS := $(PREFIX_O)_startc.o
```

Comment:
we link two .o files to crt0.o -> `arch/*/_start.S` and `_startc.c` so we still need to use `ar` (I've tried leaving _startc.o in libc - it breaks linking for ports; call to `main()` needs to be inside `crt0.o` (previously was, because crt0.o == libphoenix.a)). 

That's why I decided not to change the names right now, as renaming one of them to crt0.c/S might be confusing.

Maybe we could make  `arch/*/_start.S`  -> `crt0.o`, `_startc.c` -> `crt1.o` but for now I don't want to redesign/test solution that works as it's only a (side) step in target goal.

Reply:
Oh. Didn't spotted that. Then ok it seems to be good solution for now.